src/multibody/model.hxx

```diff
@@ -240,35 +240,35 @@ namespace se3
   inline Data::Data (const Model & ref)
     :model(ref)
```

Comment:
This has to be double checked as I am not sure of where those values are being used.